static/src/views/Chat.vue

```diff
@@ -169,22 +166,32 @@ export default {
         })
         this.$store.dispatch('update_unread_messages', unreadMessagesObj)
 
-        // TODO: refactor this code?
+        // NOTE: We are abusing the user property for these rooms. By faking 
+        // three users, we are forcing the library to always show sender / usernames 
+        // with messages. We aren't passing a real list of users because we are
+        // not currently using any of the functionality.
```

Comment:
That's right. It's by design on their side -- if it's a chat between 2 people (private or a group with just 2 people) they don't show the usernames of incoming messages, which makes sense. That's consistent with how WhatsApp works etc. I just thought for consistency it'd be nice to see the name on the incoming message for everything -- independent of whether there are 2 or 3+ people.

The hack is that the library expects us to fill out real users in that array and we'll putting in a few fake users (again, independent of whether there are more). Right now we don't use any functionality of the users array, so we're not losing anything. The comment is meant to just capture some of this in case they want to do something different in the future.